examples/conversationbot2.py

```diff
@@ -90,15 +90,15 @@ def done(bot, update, user_data):
         del user_data['choice']
 
     update.message.reply_text("I learned these facts about you:"
-                              "%s"
-                              "Until next time!" % facts_to_str(user_data))
+                              "{}"
+                              "Until next time!".format(facts_to_str(user_data)))
 
     user_data.clear()
     return ConversationHandler.END
 
 
 def error(bot, update, error):
-    logger.warn('Update "%s" caused error "%s"' % (update, error))
+    logger.warn('Update "%s" caused error "%s"', update, error)
```

Comment:
Can you change this to `logger.warning`? `Logger.warn` is deprecated.
https://docs.python.org/3/library/logging.html#logging.Logger.warning